src/libraries/System.Net.HttpListener/src/System/Net/HttpListener.cs

```diff
@@ -40,7 +40,9 @@ public HttpListener()
 
             // default: no CBT checks on any platform (appcompat reasons); applies also to PolicyEnforcement
             // config element
+#pragma warning disable CA1416 // Validate platform compatibility
             _extendedProtectionPolicy = new ExtendedProtectionPolicy(PolicyEnforcement.Never);
```

Comment:
`System.Net.Security` assembly is unsupported but seems the `ExtendedProtectionPolicy` and `PolicyEnforcement.Never` should have had supported on browser attribute, anyway suppressing for now

Reply:
@steveisok wouldn't be better to change all HttpListener like types to PNSE? That stuff does not work anyway on browser.

/cc @lewing

Reply:
@lewing This feels similar to #43387. Should we log an issue to remove the assembly-wide attribute on `System.Net.Security` and apply `[UnsupportedOSPlatform("browser")]` to all types except `ExtendedProtectionPolicy` and `PolicyEnforcement.Never`?

Reply:
re: System.Net.HttpListener, it seems like the tests were disabled for all of mono and that's why it wasn't PNSE to begin with.